README.md

```diff
@@ -372,6 +384,8 @@ benchmark:
     encryption_algo: ""
     # The value passed to '--threads' (defaults to '--auto-select-threads')
     threads: 0
+    # The value passed to '--point-in-time'
```

Comment:
nit: '--point-in-time' doesn't accept a value, this boolean indicates whether this flag should be specified or not

nit: should this be consistent with the other PiTR flags (e.g. they're pitr not point-in-time)

Reply:
Done both.